Lib/test/test_importlib/test_abc.py

```diff
@@ -357,12 +357,18 @@ def find_spec(self, fullname, path, target=None):
 
         return MetaPathSpecFinder()
 
+    def test_find_spec(self):
```

Comment:
Is nearly mimics `test_spec` with the changes you made beyond passing an explicit target. Probably worth changing the names of both test methods to more clearly define what's being tested.